src/App.jsx

```diff
@@ -84,18 +90,56 @@ function App() {
     }
 
     async function handleSignArbitraryMsg() {
-        const resp = await signClient.request({
-            chainId: CAIP_BLOCKCHAIN_ID,
-            request: {
-                method: "cosmos_sign_free_message",
-                params: {
-                    msg: msgToSign,
-                    signerAddress: address
-                }    
-            },
-            topic: session.topic
-        });
-        setSignature(resp.signature.signature);
+        const url = `${LCD_ENDPOINT}/cosmos/auth/v1beta1/accounts/${address}`;
+        const accountResponse = await fetch(url);
+        const accountData = await accountResponse.json();
+        const sequence = accountData.account.sequence;
+        const accountNumber = accountData.account.account_number;
+
+        // with amino signing
+        {
+            const sendMsg = {
+                type: "cosmos-sdk/MsgSend",
+                value: {
+                  amount: [
+                    {
+                      amount: sendAmount,
+                      denom: 'cony',
+                    },
+                  ],
+                  from_address: address,
+                  to_address: address,
+                },
+              };
+
+            const gasPrice = {amount: "0.025", denom: 'cony'}
+            const gasLimit = '100000';
+            const fee = {
+                amount: [gasPrice],
+                gas: gasLimit,
+              };
+            
+            const signDoc = makeAminoSignDoc([sendMsg], fee, CHAIN_ID, "test", accountNumber, sequence);
+
+            const params = {
+                signerAddress: address,
+                signDoc: signDoc
+            }
+            setMsgToSign(JSON.stringify(signDoc));
```

Comment:
we can make pretty json by `JSON.stringify(signDoc, null, 2);`

Reply:
Seems a bit prettier, but not much of a difference.
![스크린샷 2023-08-02 오후 5 19 37](https://github.com/dosivault/wc_v2_example/assets/55660267/b2e190cd-dcc4-4e27-9074-21ee08f32285)